persistence/src/main/scala/com/evolutiongaming/akkaeffect/persistence/PersistenceVar.scala

```diff
@@ -73,23 +75,23 @@ private[akkaeffect] object PersistenceVar {
         snapshotter: Snapshotter[F, S]
       ) = {
         actorVar.receive { persistence =>
-          def onRecoveryFailed(error: Throwable): F[Directive[persistence.Result]] = {
-            Sync[F].delay {
-              actorContext.foreach { actorContext =>
-                val msg = s"${actorContext.self.path.toStringWithoutAddress} error recovering actor state: $error"
-                actorContext.system.log.error(error, msg)
-              }
-              Directive.stop
-            }
-          }
-
           persistence
             .recoveryCompleted(seqNr, journaller, snapshotter)
             .map { result => Directive.update(result) }
             .handleErrorWith(onRecoveryFailed)
         }
       }
 
+      def onRecoveryFailed[A](error: Throwable): F[Directive[A]] = {
```

Comment:
could we pass a `name` of function as argument to be able to trace failure in the future?
like

```$name failed: $error```

where name is one of `snapshotOffer`, `event` or `recoveryCompleted`